packages/tokens/build/ios/StyleDictionarySize.swift

```diff
@@ -0,0 +1,73 @@
+
+//
+// StyleDictionarySize.swift
+//
+
+//
+
+import UIKit
+
+public enum StyleDictionarySize {
+    public static let SizeFont01 = 10.90f
+    public static let SizeFont02 = 12.00f
+    public static let SizeFont03 = 13.00f
+    public static let SizeFont04 = 14.00f
+    public static let SizeFont05 = 16.00f
+    public static let SizeFont06 = 18.00f
+    public static let SizeFont07 = 23.00f
+    public static let SizeFont08 = 28.00f
+    public static let SizeFont09 = 34.00f
+    public static let SizeFont10 = 41.00f
+    public static let SizeFont11 = 49.00f
+    public static let SizeFont12 = 59.00f
+    public static let SizeGutterScreenM = 32.00f
+    public static let SizeGutterScreenS = 16.00f
+    public static let SizeLine01L = 16.00f
+    public static let SizeLine01M = 14.00f
+    public static let SizeLine01S = 12.00f
+    public static let SizeLine01Xs = 12.00f
+    public static let SizeLine02L = 18.00f
+    public static let SizeLine02M = 16.00f
+    public static let SizeLine02S = 14.00f
+    public static let SizeLine02Xs = 12.00f
+    public static let SizeLine03L = 20.00f
+    public static let SizeLine03M = 18.00f
+    public static let SizeLine03S = 16.00f
+    public static let SizeLine03Xs = 14.00f
+    public static let SizeLine04L = 22.00f
+    public static let SizeLine04M = 18.00f
+    public static let SizeLine04S = 16.00f
+    public static let SizeLine04Xs = 14.00f
+    public static let SizeLine05L = 24.00f
+    public static let SizeLine05M = 22.00f
+    public static let SizeLine05S = 18.00f
+    public static let SizeLine05Xs = 16.00f
+    public static let SizeLine06L = 28.00f
+    public static let SizeLine06M = 24.00f
+    public static let SizeLine06S = 20.00f
+    public static let SizeLine06Xs = 18.00f
+    public static let SizeLine07L = 36.00f
+    public static let SizeLine07M = 32.00f
+    public static let SizeLine07S = 28.00f
+    public static let SizeLine07Xs = 24.00f
+    public static let SizeLine08L = 44.00f
+    public static let SizeLine08M = 36.00f
+    public static let SizeLine08S = 32.00f
+    public static let SizeLine08Xs = 28.00f
+    public static let SizeLine09L = 52.00f
+    public static let SizeLine09M = 44.00f
+    public static let SizeLine09S = 40.00f
+    public static let SizeLine09Xs = 36.00f
+    public static let SizeLine10L = 60.00f
+    public static let SizeLine10M = 56.00f
+    public static let SizeLine10S = 48.00f
+    public static let SizeLine10Xs = 40.00f
+    public static let SizeLine11L = 72.00f
+    public static let SizeLine11M = 68.00f
+    public static let SizeLine11S = 56.00f
+    public static let SizeLine11Xs = 48.00f
+    public static let SizeLine12L = 88.00f
+    public static let SizeLine12M = 80.00f
+    public static let SizeLine12S = 68.00f
+    public static let SizeLine12Xs = 60.00f
```

Comment:
Float suffix does not exist in Swift 
```suggestion
    public static let SizeFont01 = 10.90
    public static let SizeFont02 = 12.00
    public static let SizeFont03 = 13.00
    public static let SizeFont04 = 14.00
    public static let SizeFont05 = 16.00
 ...
```